fastasyncworldedit/API/fill-region-by-pattern.md

```diff
@@ -0,0 +1,78 @@
+# Filling a region using a pattern
+
+## Foreword
+First of all, patterns usually come from the core part of Worleedit or FAWE. The patterns can be found in the following path: [worldedit-core/src/main/java/com/fastasyncworldedit/core/function/pattern](https://intellectualsites.github.io/fastasyncworldedit-javadocs/worldedit-core/com/fastasyncworldedit/core/extension/factory/parser/pattern/package-summary.html)
```

Comment:
Those are the pattern parsers used for commands. It should rather link https://intellectualsites.github.io/fastasyncworldedit-javadocs/worldedit-core/com/sk89q/worldedit/function/pattern/Pattern.html, where all known implementing classes are listed. I would word it something like `Existing patterns are listed [here](...)`.